$NetBSD: patch-ab,v 1.5 2014/03/10 13:01:17 fhajny Exp $

enable POSIX functionality by default
set the logging paths
set up the pid files

--- prosody.cfg.lua.dist.orig	2014-01-12 11:41:40.000000000 +0000
+++ prosody.cfg.lua.dist
@@ -63,7 +63,7 @@ modules_enabled = {
 		--"http_files"; -- Serve static files from a directory over HTTP
 
 	-- Other specific functionality
-		--"posix"; -- POSIX functionality, sends server to background, enables syslog, etc.
+		"posix"; -- POSIX functionality, sends server to background, enables syslog, etc.
 		--"groups"; -- Shared roster support
 		--"announce"; -- Send announcement to all online users
 		--"welcome"; -- Welcome users who register accounts
@@ -139,12 +139,15 @@ authentication = "internal_plain"
 -- Logging configuration
 -- For advanced logging see http://prosody.im/doc/logging
 log = {
-	info = "prosody.log"; -- Change 'info' to 'debug' for verbose logging
-	error = "prosody.err";
+	info = "@PROSODY_LOG@/prosody.log"; -- Change 'info' to 'debug' for verbose logging
+	error = "@PROSODY_LOG@/prosody.err";
 	-- "*syslog"; -- Uncomment this for logging to syslog
 	-- "*console"; -- Log to the console, useful for debugging with daemonize=false
 }
 
+-- Enable PID file by default
+pidfile = "@PROSODY_RUN@/prosody.pid";
+
 ----------- Virtual hosts -----------
 -- You need to add a VirtualHost entry for each domain you wish Prosody to serve.
 -- Settings under each VirtualHost entry apply *only* to that host.
